sled-agent/src/rack_setup/plan/service.rs

```diff
@@ -387,6 +389,21 @@ impl Plan {
                     };
                     sled_info.request.datasets.insert(config.id, config);
                 }
+
+                // LocalStorage isn't in the U2_EXPECTED_DATASETS list, add it
+                // here. XXX does RSS blueprint have to be blippy clean? Isn't
+                // the plan to RSS only enough to run Nexus and have it use
+                // blueprints to make the rest true?
```

Comment:
> XXX does RSS blueprint have to be blippy clean?

Yes - blippy should only complain about blueprints that should not exist and may lead to some kind of internal inconsistencies, I think?

> Isn't the plan to RSS only enough to run Nexus and have it use blueprints to make the rest true?

That is one possible plan for a future where we've rewritten RSS. :sweat_smile: Today, we expect the Nexus takeover after RSS to find that it does not need to make any changes to the current target blueprint (and I think we even have a test for that somewhere - @davepacheco wrote it IIRC).

Reply:
:+1: rewrote this comment in e8b0d009006161c28bf462cab64a94c1ebde790d